app/controllers/comments_controller.rb

```diff
@@ -1,5 +1,40 @@
 class CommentsController < ApplicationController
-  def index; end
+  before_action :set_comment, only: %i[index show update destroy]
+
+  def index
+    @comments = Comment.all
+  end
+
+  def edit; end
 
   def show; end
+
+  def new
+    puts 'I am in this method'
+    @comment = Comment.new
+  end
+
+  def create
+    user = User.find(params[:user_id])
+    post = Post.find(params[:post_id])
+    created_comment = Comment.new(params.require(:comment).permit(:text))
+    created_comment.authorId = user.id
+    created_comment.postId = post.id
+    if created_comment.save
```

Comment:
- [ ] Comments should be created on behalf of the currently logged in user (current_user) of the app.
Hint you already created current_user in your application_controller so you can take advantage of that 👍 

- [ ] Kindly implement and use strong params while creating new comments. The reason for this is because it increases the security of the data sent through your forms by allowing only what has been explicitly permitted. See a lesson on its usage [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) 😎

Reply:
This issue is well resolved! ✔️